pkg/daemon/rpm-ostree.go

```diff
@@ -100,6 +115,60 @@ func (r *RpmOstreeClient) loadStatus() (*rpmOstreeState, error) {
 	return &rosState, nil
 }
 
+func parseVer(s string) ([]int, error) {
```

Comment:
```suggestion
func parseRpmOstreeVer(s string) ([]int, error) {
```

can you also add a docstring to this function bc it's really not clear what it is doing.